quinn-proto/src/connection.rs

```diff
@@ -1041,13 +1016,11 @@ impl Connection {
                             }
                             match frame {
                                 Frame::Padding => {}
-                                Frame::Stream(
-                                    frame @ frame::Stream {
-                                        id: StreamId(0), ..
-                                    },
-                                ) => self.read_tls(&frame),
+                                Frame::Crypto(frame) => {
+                                    self.read_tls(&frame);
```

Comment:
I think these need to be assembled first, right?

Reply:
`Connection::read_tls` performs assembly.